app/helpers/katalyst/content/frontend_helper.rb

```diff
@@ -3,9 +3,18 @@
 module Katalyst
   module Content
     module FrontendHelper
+      # Render all items from a content version as HTML
+      # @param version [Katalyst::Content::Version] The content version to render
+      # @return [ActiveSupport::SafeBuffer,String,nil] Content as HTML
+      # Example usage:
+      #   <%= render_content(version) %>
       def render_content(version)
-        without_partial_path_prefix do
-          render partial: version.tree.select(&:visible?)
+        capture do
+          cache version.parent do
```

Comment:
I don't think this approach is quite right because the items are supposed to be immutable. We should be able to cache the generated HTML for each version (i.e. `cache version`)

Reply:
See `NavigationHelper` for a example of how it's implemented for navigation.

```
cache menu.published_version do
    concat builder.render(menu.published_tree)
end
```